src/pd/metrics.rs

```diff
@@ -20,4 +20,32 @@ lazy_static! {
             "Bucketed histogram of PD message send duration",
              exponential_buckets(0.0005, 10.0, 7).unwrap()
         ).unwrap();
+
+    pub static ref PD_REQ_COUNTER_VEC: CounterVec =
+        register_counter_vec!(
+            "tikv_raftstore_pd_request_sent_total",
```

Comment:
Now, it does not belong to `tikv_raftstore_` any more.

Reply:
+1
We should also update the grafana too.